tests/conftest.py

```diff
@@ -222,17 +222,24 @@ def _framework_fixture_inner(request, app, slug, **kwargs):
 
 
 def _supplierframework_fixture_inner(request, app, sf_kwargs=None):
+    """
```

Comment:
Well, it does a job in an interesting way (i.e. I wouldn't have thought of doing it like this).